src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClientX/IO/TdsStream.cs

```diff
@@ -77,39 +80,41 @@ public void ReplaceUnderlyingStream(Stream stream)
         /// <exception cref="NotImplementedException"></exception>
         public void SetPacketSize(int packetSize)
         {
-            _writeStream.SetPacketSize(packetSize);
-            // TODO: Do this for the read stream as well.
+            throw new NotImplementedException();
+        }
+
+        /// <summary>
+        /// When writing the packet, the caller needs to 
+        /// specify the packet type. 
+        /// TODO: Consider accepting an enum of packet types
```

Comment:
This is already done.

Reply:
True. Will fix in next commit

Reply:
We dont need this method at all. There is a Property `public TdsStreamPacketType? PacketHeaderType ` which does this. I will delete this method.